google-cloud-spanner/pom.xml

```diff
@@ -47,9 +47,10 @@
         </configuration>
         <executions>
           <execution>
+            <!-- Executes unit tests -->
             <id>default-test</id>
             <configuration>
-              <excludedGroups>com.google.cloud.spanner.TracerTest,com.google.cloud.spanner.IntegrationTest</excludedGroups>
+              <excludedGroups>com.google.cloud.spanner.categories.TracerTest,com.google.cloud.spanner.categories.IntegrationTest</excludedGroups>
```

Comment:
You probably need to change the package name for `TracerTest` further below as well (line 62)

Reply:
Ah yes, thanks, fixed it.